### ISSUE TITLE
flux run segfaults if user is not in password file on compute node

### ISSUE THREAD

**Issue by garlick** [flux-framework/flux-core#5466]
Eh I was unable to confirm what was going on before my user was created on the compute node.  This is just a note to go back and try this.  Note this is 
```
[garlick@elcap1:~]$ flux version
commands:               0.53.0
libflux-core:           0.53.0
libflux-security:       0.9.0
build-options:          +ascii-only+systemd+hwloc==2.8.0+zmq==4.3.4
```

flux dmesg says
```
2023-09-22T17:04:38.839594Z job-manager.info[0]: f9Tfk8J13oV: prolog: stdout: /etc/flux/system/prolog-user-scripts.d/setup_x11.sh: exited with exit code 1
2023-09-22T17:04:38.841400Z job-manager.err[0]: f9Tfk8J13oV: prolog: stderr: elcap54 (rank 11): sudo: unknown user: #5588
```

I'm not sure what segfaulted or if something segfaulted remotely and I just got that in my output.

**Comment by grondo** [comment]
Is this just perhaps flux-framework/flux-security#170?

**Comment by garlick** [comment]
Oh duh.  I thought that seemed familiar.  That was fixed in 0.10 and they've got 0.9 installed.

Since I now recall that I tested this on my cluster and that things behaved reasonably with myself not in the password file on one of the nodes, I'll close this.  Thanks for remembering that!
